.dependabot/config.yml

```diff
@@ -0,0 +1,11 @@
+version: 1
+update_configs:
+  - package_manager: "go:modules"
+    directory: "/"
+    target_branch: "legacy-1-15"
+    update_schedule: "daily"
```

Comment:
I think even less frequent would be fine.

Reply:
This will be removed from this PR as it is already on master anyway. But we should at least get checked for security updates daily. That is also configurable on master